cloudify_agent/installer/__init__.py

```diff
@@ -350,6 +350,10 @@ def upload_certificate(self):
         cert_content = self.rest_cert_content
 
         if not os.path.exists(cert_file):
+            cert_file = os.path.expanduser(cert_file)
```

Comment:
it is logically impossible for exists to return True and at the same time have expanduser have any affect